src/config/2022.json

```diff
@@ -1281,6 +1281,14 @@
       ],
       "github":"yokoka"
     },
+    "Levix":{
+      "name": "Zhiwei Li",
+      "teams":[
+        "translators"
+      ],
+      "avatar_url": "65117011",
+      "github":"Levix"
+    },
```

Comment:
@Levix can you confirm you're happy with this?

You can add Twitter, Personal website, or LinkedIn usernames too if you want.

Reply:
Yeah.